### ISSUE TITLE
pip install error

### ISSUE THREAD

**Issue by aaronsgithub** [jmcarp/betfair.py#29]
Using pip 6.1.1, Python 3.4.3 I get the following error:

```
$ pip install betfair.py
Collecting betfair.py
  Downloading betfair.py-0.1.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/user/1000/pip-build-8mzy35ag/betfair.py/setup.py", line 79, in <module>
        license=read('LICENSE'),
      File "/tmp/user/1000/pip-build-8mzy35ag/betfair.py/setup.py", line 61, in read
        with open(fname) as fp:
    FileNotFoundError: [Errno 2] No such file or directory: 'LICENSE'
```

**Comment by goetzk** [comment]
This looks like it was fixed in PR #31

**Comment by jmcarp** [comment]
Yep. Closing.
